libs/bixarena/api-description/src/components/schemas/VotePage.yaml

```diff
@@ -0,0 +1,12 @@
+type: object
+description: A page of votes
+properties:
+  data:
+    type: array
+    items:
+      $ref: Vote.yaml
+  page:
+    $ref: PageMetadata.yaml
+required:
+  - data
+  - page
```

Comment:
Do we need to expose list of votes outside of the API service? (see comment below)